### PR TITLE
#854. Deleting means of appeal and judgment results after import process job

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java [hunk 1]
```diff
@@ -67,4 +78,46 @@ public void deleteJudgmentsWithoutRawSourceJudgment(Class<? extends Judgment> ju
         }
     }
     
+    @Transactional
+    public void deleteMeansOfAppealWithoutJudgments(CourtType courtType) {
```
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: In my view the better idea would be to put these 2 new methods into a separate service as it has been done in the case of deleting sc objects. See: ScObjectDeleter.

Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: Commet missing

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java [hunk 2]
```diff
@@ -67,4 +78,46 @@ public void deleteJudgmentsWithoutRawSourceJudgment(Class<? extends Judgment> ju
...
+                .getResultList();
+        
+        meansOfAppealIds.stream().forEach(meansOfAppealRepository::delete);
+        
+        meansOfAppealRepository.flush();
+        
+    }
+    
+    @Transactional
+    public void deleteJudgmentResultsWithoutJudgments(CourtType courtType) {
```
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: comment missing

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java (hunk 1 above)
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: I wonder if we shouldn't move this service to the import module. The same as it has been done with ScObjectDeleter.

Comment (madryk) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: comment: :+1: 
move to import module: it already is in import module

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java (hunk 2 above)
Comment (madryk) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: :+1:

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java (hunk 1 above)
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java]: your're right, sorry.

**Pull request closed without merging**